lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -435,6 +436,15 @@ func (c *Client) startInputStreaming(stopCh chan struct{}) error {
 			return err
 		}
 
+		if m, ok := msg.(tdp.Ping); ok {
```

Comment:
Do we need a ping message at all? What if the agent just did periodic latency checks and reported its status by injecting a `LatencyStats` message in the stream?

Reply:
We need it for measuring the latency between proxy and WDS, otherwise we will measure misleading latency where WDS is close to the desktop but far away from the proxy